core/templates/services/UpgradedServices.ts

```diff
@@ -687,8 +687,10 @@ export class UpgradedServices {
   getUpgradedServices(): UpgradedServicesDict {
     var upgradedServices = {};
     // We are using eslint disable here for multilines because we have not used
-    // dot notation at a lot of places so it is not possible to use
-    // eslint disable next line for each of them.
+    // dot notation at a lot of places so it does not seem practical to use
+    // "eslint disable next line" for each of them and we cant use dot notation
```

Comment:
for each of them. Also, we can't use dot notation either because...

(Note also: spelling of "can't", "because")

Reply:
Done.